### ISSUE TITLE
Unable to add a transaction to a batch because of a Currency Type error

### ISSUE THREAD

**Issue by BEGamache** [SparkDevNetwork/Rock#3905]
### Prerequisites
* [x] Put an X between the brackets on this line if you have done all of the following:
    * Can you reproduce the problem on a fresh install or the [demo site](http://rock.rocksolidchurchdemo.com/)?
    * Did you include your Rock version number and [client culture](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?
    * Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ASparkDevNetwork%2FSlack) to see if your bug or enhancement is already reported?

### Description

We are unable to add a transaction to a batch, from the internal site, because selecting a Currency Type from the drop down throws an error.  The error goes away when the blank option is selected but when cash, check, or any of the other types is selected the error below is thrown. 

```
Error
53|error|500|Object reference not set to an instance of an object.|
```

I tested this on the the demo server (v9.1) and did not have this issue. I went through the commit history for this block and it has something to do with the commit on Aug 26, 2019.
https://github.com/SparkDevNetwork/Rock/commit/d3e1dccc2b0197af49972abfcc3a68b68a5f58d9#diff-a1a05f6627a73e569dfbe66d60bf6253

The code before this commit works and doesn't throw the error. When I update the block's code to the **d3e1dcc** commit I get the error.

### Steps to Reproduce

1. Go to the Batches page, create a batch, then create a transaction in that batch
2. Select a Currency Type
3. Error will display above

**Expected behavior:**

Select a currency type without the error occurring

**Actual behavior:**

Currency Type is required but when a currency type is selected, an error is thrown.

### Versions

* **Rock Version:** We're on 9.2, Demo is on 9.1
* **Client Culture Setting:** en-US

**Comment by cabal95** [comment]
I believe this is actually a duplicate of #3894.  @BEGamache Can you verify if your currency type of "Non-Cash" was deleted?

**Comment by BEGamache** [comment]
Non-Cash is still present.

<img width="478" alt="Screen Shot 2019-09-18 at 1 25 56 PM" src="https://user-images.githubusercontent.com/5738650/65175180-fde11380-da17-11e9-9beb-bcda03f2d833.png">

<img width="691" alt="Screen Shot 2019-09-18 at 1 27 12 PM" src="https://user-images.githubusercontent.com/5738650/65175222-13eed400-da18-11e9-8627-4a3585dd2503.png">

**Comment by JimMichael** [comment]
I can confirm this exception on my 9.2 test copy of prod database
![image](https://user-images.githubusercontent.com/3903408/65176459-6af5a880-da1a-11e9-82ac-4fb9b97e72cc.png)

**Comment by JimMichael** [comment]
(We also have Non-Cash currency type, though for all I know it migrated from Arena and doesn't have the same GUID as stock Rock

**Comment by JimMichael** [comment]
Because I had a snapshot for 9.1 I could easily revert to, I did so and the above works fine there. So it is indeed something that has cropped up between 9.1 and 9.2.

**Comment by cabal95** [comment]
@BEGamache Okay, it looks like this is the same problem then. The below should be the GUID (and query to find the stock one).

@JimMichael can you run the below query to verify that yours is missing? If so I think we can close this in favor of the original (which has a link back to this issue already).

```sql
select * from DefinedValue WHERE [Guid] = '7950ff66-80ee-e8ab-4a77-4a13edeb7513'
```

**Comment by JimMichael** [comment]
@cabal95 Yep, no record returned with that query.

**Comment by cabal95** [comment]
Great thanks. Closing this one. The other was also just closed as fixed, so I'm guessing 9.3 will have the fix.
